_posts/2022-04-30-keeping-up-with-carpentrycon2022.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title:  "Keeping up with CarpentryCon2022"
+date:   2022-04-30 09:00:00
```

Comment:
Do we need to update this date?

Reply:
I wanted this to sort in the list before the proposal submission post, so the postdate was my workaround.

Reply:
Gotcha!

Reply:
I'd be open to a better solution ... but reached my limits of knowledge of Jekyll on this one :)